index.js

```diff
@@ -62,7 +62,7 @@ module.exports = class MongooseTrailpack extends DatastoreTrailpack {
         schema.statics = model.statics
         schema.methods = model.methods
 
-        model.onSchema(schema)
+        model.onSchema(schema, this.app)
```

Comment:
I prefer breaking retro compatibility and have unify interfaces between ORM. We'll update the module with a minor version not a patch. If we don't we will have if(mongoose) invert stuff in all trailpack that want to be multi ORM.